File: C#/AnaliseDeSentimento/AnaliseDeSentimento/Arquivo/LerArquivo.cs
using System.Collections.Generic;
using System.IO;

namespace AnaliseDeSentimento.Arquivo
{
    public class LerArquivo
    {
        public string NomeArquivo { get; private set; }
        public int QuantidadeLinhas { get; private set; }
        public List<string> Linhas { get; private set; }

        public LerArquivo(string nomeArquivo)
        {
            NomeArquivo = nomeArquivo;
            Linhas = new List<string>();

            IniciarLeitura();
        }

        void IniciarLeitura()
        {
            StreamReader file = new(NomeArquivo);
            string linhaAtual = file.ReadLine();

            if (int.TryParse(linhaAtual, out int quantidadeLinhas))
            {
                QuantidadeLinhas = quantidadeLinhas;
                ExecutarLeitura(file);
            }
        }

        void ExecutarLeitura(StreamReader file)
        {
            for (int i = 0; i < QuantidadeLinhas; i++)
                Linhas.Add(file.ReadLine());
        }
    }
}

File: Python/Arquivo/LerArquivo.py
class LerArquivo:
  def __init__(self, nomeArquivo):
    self.NomeArquivo = nomeArquivo
    self.QuantidadeLinhas = 0
    self.Linhas = []
    self.IniciarLeitura()

  def IniciarLeitura(self):
    refArquivo = open(self.NomeArquivo, "r")

    linhaAtual = refArquivo.readline()
    self.QuantidadeLinhas = int(linhaAtual)
    self.ExecutarLeitura(refArquivo)

    refArquivo.close()

  def ExecutarLeitura(self, refArquivo):
    for index in range(self.QuantidadeLinhas):
      linha = refArquivo.readline().replace('\n', '')
      
      self.Linhas.append(linha)
